.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: java
+
+sudo: required
+
+services:
+  - docker
+
+script:
+  - make bin
```

Comment:
quick note- let's just run make.  make bin does not actually compile the sdk, it uses a prebuilt jar.